hibernate-core/src/test/java/org/hibernate/orm/test/mapping/lazytoone/Flight.java

```diff
@@ -25,7 +24,6 @@
 public class Flight {
 	@Id
 	private Integer id;
-	@Column(name = "flight_number")
```

Comment:
Please revert this change. That causes an issue on Oracle as `number` is a keyword.

Reply:
+1 - not sure why I changed that